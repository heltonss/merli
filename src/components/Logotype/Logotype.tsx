import Logo from 'assets/images/Logo_ML.png';
import Logo2x from 'assets/images/Logo_ML@2x.png';
import React from 'react';

interface IOwnProps {
  parentClassStyle?: string;
}

const Logotype = ({ parentClassStyle }: IOwnProps) => {
  return (
    <img
      className={parentClassStyle}
      src={Logo}
      alt="logotipo mercado livre"
      srcSet={`${Logo2x} 2x`}
    />
  );
};

export default Logotype;
