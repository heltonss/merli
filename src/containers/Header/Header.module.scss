@import '../../assets/scss/colors';
@import '../../assets//scss//screen';

.container {
  background: $color-header;
  align-items: center;
  display: flex;
  width: 100%;
  height: 70px;
  @media only screen {
    @media (min-width: $screen-md) {
      & {
        padding: 0 11%;
      }
    }
  }
}

.logotype {
  display: block;
  margin-right: 23px;
}
