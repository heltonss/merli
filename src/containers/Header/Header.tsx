import Logotype from 'components/Logotype/Logotype';
import Search from 'components/Search/Search';
import React, { Component } from 'react';
import Style from './Header.module.scss';

export class Header extends Component {
  render() {
    return (
      <div className={Style.container}>
        <div className={Style.containerSafe}>
          <Logotype parentClassStyle={Style.logotype} />
          <Search />
        </div>
      </div>
    );
  }
}

export default Header;
