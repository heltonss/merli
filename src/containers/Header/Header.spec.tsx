import Logotype from 'components/Logotype/Logotype';
import { shallow } from 'enzyme';
import React from 'react';
import Header from './Header';

describe('Header', () => {
  it('Should render withou crash', () => {
    const wrapper = shallow(<Header />);

    expect(wrapper).toMatchSnapshot();
  });

  it('Should have logo mercado livre', () => {
    const wrapper = shallow(<Header />);

    expect(wrapper.find(Logotype)).toHaveLength(1);
  });
});
