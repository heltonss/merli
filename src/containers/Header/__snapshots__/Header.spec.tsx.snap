// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should render withou crash 1`] = `ShallowWrapper {}`;
